compiler/accelerators/snax.py

```diff
@@ -152,9 +152,15 @@ def _generate_streamer_setup_vals(
                 cst = arith.Constant.from_int_and_width(stride.data, i32)
                 result.append(([cst], cst.result))
 
+            # channel mask option
+            if StreamerOpts.HasChannelMask in streamer.opts:
+                # default to 0 for now
+                c0 = arith.Constant.from_int_and_width(0, i32)
+                result.append(([c0], c0.result))
+
```

Comment:
Can you also update the SNAX version for this feature in this PR?

Reply:
that happens in #264 as there are a lot of unrelated changes which I wanted to separate